talk/owt/sdk/base/globalconfiguration.cc

```diff
@@ -21,5 +21,9 @@ AudioProcessingSettings GlobalConfiguration::audio_processing_settings_ = {
 AudioProcessingSettings GlobalConfiguration::audio_processing_settings_ = {
     true, true, true, true};
 #endif
+#ifdef OWT_ENABLE_QUIC
```

Comment:
Please move it to ConferenceClientConfiguration since it could be configured per client. And we don't want too many things to be globally shared.

Reply:
done.